### PR TITLE
agpl/imagemagick variants

### PR DESCRIPTION
**Pull request by jonashaag**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jonashaag** [comment]
@conda-forge-admin, please rerender

**Comment by jonashaag** [comment]
@conda-forge-admin, please rerender

**Comment by jonashaag** [comment]
```
config.status: error: Something went wrong bootstrapping makefile fragments
    for automatic dependency tracking.  If GNU make was not used, consider
    re-running the configure script with MAKE="gmake" (or whatever is
    necessary).  You can also try re-running configure with the
    '--disable-dependency-tracking' option to at least be able to build
    the package (albeit without support for automatic dependency tracking).
See `config.log' for more details
```

seems unrelated

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,24 +13,26 @@ source:
...
   host:
-    - make  # [linux]
+    - make
     - bzip2
     - fftw
     - fontconfig
     - fonts-conda-forge
     - freetype
-    - ghostscript
+    - ghostscript  # [license_family=="agpl"]
```
Comment (ocefpaf) [inline recipe/meta.yaml:35]: @isuruf  comment on matrix that this should be on build only. Sorry but I merged before I saw the comment.

**Pull request merged**